File: gcc/testsuite/gcc.dg/gomp/appendix-a/a.1.1.c
/* { dg-do compile } */

void
a1 (int n, float *a, float *b)
{
  int i;
#pragma omp parallel for
  for (i = 1; i < n; i++)	/* i is private by default */
    b[i] = (a[i] + a[i - 1]) / 2.0;
}

File: gcc/testsuite/gfortran.dg/gomp/appendix-a/a.1.1.f90
! { dg-do compile }
      SUBROUTINE A1(N, A, B)
      INTEGER I, N
      REAL B(N), A(N)
!$OMP PARALLEL DO !I is private by default
      DO I=2,N
          B(I) = (A(I) + A(I-1)) / 2.0
      ENDDO
!$OMP END PARALLEL DO
      END SUBROUTINE A1
